integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/AnsibleSteps.java

```diff
@@ -0,0 +1,52 @@
+package com.redhat.service.smartevents.integration.tests.steps;
+
+import java.time.Duration;
+
+import org.assertj.core.api.Assertions;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.Assumptions;
+
+import com.redhat.service.smartevents.integration.tests.context.TestContext;
+import com.redhat.service.smartevents.integration.tests.context.resolver.ContextResolver;
+import com.redhat.service.smartevents.integration.tests.resources.AnsibleResource;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+
+public class AnsibleSteps {
+
+    private TestContext context;
+
+    public AnsibleSteps(TestContext context) {
+        this.context = context;
+    }
+
+    @Given("^Ansible instance url was set$")
+    public void checkAnsibleUrlWasSet() {
+        final String ansibleEndpoint = System.getProperty("ansible.endpoint");
```

Comment:
You can enforce providing system property by using `Utils.getSystemProperty`, that way you can avoid this step, making sure that the property is set.

Reply:
Makes me think, how to distinguish whether the test should be executed in the environment and `ansible.endpoint` wasn't accidentally set, or whether it wasn't set because the test should be skipped.

IMHO better approach is to tag the test and exclude the tag in environments where the test should be skipped, i.e. https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/.github/workflows/dev-cluster-check.yml#L57.